### ISSUE TITLE
Any news from devs ?

### ISSUE THREAD

**Issue by Hetan-Layn** [Jessecar96/SteamDesktopAuthenticator#1057]
Is there a chance this app will be updated soon?
Is this project dead ?

**Comment by dyc3** [comment]
Yes, this project is effectively dead. You should just go use the steam mobile app if you can.
